cypress/e2e/settings/users/patron-block/patron-block-maximum-items-overdue-removed-after-item-renewed.cy.js

```diff
@@ -0,0 +1,253 @@
+      })
+      .then(() => {
+        UserEdit.addServicePointViaApi(testData.userServicePoint.id, userData.userId, testData.userServicePoint.id);
+
+        cy.get('@items').each((item) => {
+          Checkout.checkoutItemViaApi({
+            id: uuid(),
+            itemBarcode: item.barcode,
+            loanDate: moment.utc().format(),
+            servicePointId: testData.userServicePoint.id,
+            userBarcode: userData.barcode,
+          });
+        });
+
+        cy.login(userData.username, userData.password);
+      });
+  });
+
+  after('Deleting created entities', () => {
+    cy.get('@items').each((item) => {
+      CheckInActions.checkinItemViaApi({
+        itemBarcode: item.barcode,
+        servicePointId: testData.userServicePoint.id,
+        checkInDate: new Date().toISOString(),
+      });
+    });
+    PaymentMethods.deleteViaApi(testData.paymentMethodId);
+    UsersOwners.deleteViaApi(owner.data.id);
+    cy.deleteLoanPolicy(loanPolicyBody.id);
+    UserEdit.changeServicePointPreferenceViaApi(userData.userId, [testData.userServicePoint.id]);
+    ServicePoints.deleteViaApi(testData.userServicePoint.id);
+    Users.deleteViaApi(userData.userId);
+    PatronGroups.deleteViaApi(patronGroup.id);
+    cy.get('@items').each((item, index) => {
+      cy.deleteItem(item.itemId);
+      cy.deleteHoldingRecordViaApi(itemsData.itemsWithSeparateInstance[index].holdingId);
+      InventoryInstance.deleteInstanceViaApi(itemsData.itemsWithSeparateInstance[index].instanceId);
+    });
+    Conditions.resetConditionViaApi('584fbd4f-6a34-4730-a6ca-73a6a6a9d845', 'Maximum number of overdue items');
+    Location.deleteViaApiIncludingInstitutionCampusLibrary(
+      testData.defaultLocation.institutionId,
+      testData.defaultLocation.campusId,
+      testData.defaultLocation.libraryId,
+      testData.defaultLocation.id
+    );
+    CirculationRules.deleteRuleViaApi(originalCirculationRules);
+    cy.deleteLoanType(testData.loanTypeId);
+  });
+  it(
+    'C350654 Verify automated patron block "Maximum number of overdue items" removed after overdue item renewed (vega)',
+    { tags: [TestTypes.criticalPath, devTeams.vega] },
+    () => {
+      cy.visit(SettingsMenu.conditionsPath);
+      Conditions.waitLoading();
+      Conditions.select('Maximum number of overdue items');
+      Conditions.setConditionState(blockMessage);
+      cy.visit(SettingsMenu.limitsPath);
+      Limits.selectGroup(patronGroup.name);
+      Limits.setMaximumNumberOfOverdueItems('4');
+      // needed for the "Loan Policy" so item can get overdue status
+      cy.wait(120000);
+
+      cy.visit(TopMenu.usersPath);
+      UsersSearchPane.waitLoading();
+      UsersSearchPane.searchByKeywords(userData.barcode);
+      UsersCard.waitLoading();
+      Users.checkIsPatronBlocked(blockMessage, 'Borrowing, Renewals, Requests');
+
+      const itemForCheckIn = itemsData.itemsWithSeparateInstance[0];
+      UsersCard.openLoans();
```

Comment:
The test is failing in this method sometimes. Please update the waiter in this method:  cy.wait('@getLoans', { requestTimeout: 10000 });

Reply:
done